backend/app/services/knowledge/permission_service.py

```
@@ -0,0 +1,786 @@
+        Get user's permission for a knowledge base.
+
+        Returns:
+            Tuple of (has_access, permission_level, is_creator)
+        """
+        # Get the knowledge base
+        kb = (
+            db.query(Kind)
+            .filter(
+                Kind.id == knowledge_base_id,
+                Kind.kind == "KnowledgeBase",
+                Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)